src/Tickets/Emails/Admin/Preview_Modal.php

```diff
@@ -189,7 +189,8 @@ public function get_modal_content_ajax( $render_response, $vars ) {
 		$ticket_bg_color = Arr::get( $vars, 'ticketBgColor', '' );
 
 		if ( ! empty( $ticket_bg_color ) ) {
-			$preview_context['ticket_bg_color'] = wp_kses( $ticket_bg_color, [] );
+			$preview_context['ticket_bg_color']   = wp_kses( $ticket_bg_color, [] );
+			$preview_context['ticket_text_color'] = \Tribe__Utils__Color::get_contrast_color( $preview_context['ticket_bg_color'] );
```

Comment:
Since we're wrapping the bg color in `wp_kses()`, we should probably do the same with the text color.

Reply:
This one and the other one below

Reply:
@codingmusician we are re-using the var from the line above where it was already sanitized.